src/Altinn.Profile.Integrations/Migration/v0.06/01-add-index.sql

```diff
@@ -0,0 +1 @@
+CREATE UNIQUE INDEX ix_notifications_address_registry_id ON organization_notification_address.notifications_address (registry_id);
```

Comment:
We can add a check like this:
```suggestion
CREATE UNIQUE INDEX IF NOT EXISTS ix_notifications_address_registry_id ON organization_notification_address.notifications_address (registry_id);
```
Ideally unnecessary, but it a safety net.